mmv1/products/networkservices/WasmPlugin.yaml

```diff
@@ -0,0 +1,203 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'WasmPlugin'
+description: |
+  WasmPlugin is a resource representing a service executing a customer-provided Wasm module.
+references:
+  guides:
+    'Configure a route extension': 'https://cloud.google.com/service-extensions/docs/create-plugin'
+  api: 'https://cloud.google.com/service-extensions/docs/reference/rest/v1/projects.locations.wasmPlugins'
+docs:
+base_url: 'projects/{{project}}/locations/{{location}}/wasmPlugins'
+self_link: 'projects/{{project}}/locations/{{location}}/wasmPlugins/{{name}}'
+create_url: 'projects/{{project}}/locations/{{location}}/wasmPlugins?wasmPluginId={{name}}'
+create_verb: 'POST'
+update_verb: 'PATCH'
+update_mask: true
+read_query_params: '?view=WASM_PLUGIN_VIEW_FULL'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+autogen_async: true
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+  result:
+    resource_inside_response: false
+custom_code:
+examples:
+  - name: 'wasm_plugin_basic_docker_repository'
```

Comment:
Why are these examples skipped for testing? I'm seeing failures when trying to run them